rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import babel from 'rollup-plugin-babel';
+
+export default {
+  entry: 'src/TextareaAutosize.js',
+  dest: 'lib/TextareaAutosize.min.js',
```

Comment:
destination file should be put to `dist` rather than to `lib` + we could built both minified and unminified versions of the file (based on `NODE_ENV` ofc)